helm/aws-crossplane-cluster-config-operator/README.md

```diff
@@ -0,0 +1,79 @@
+
+Note that configuration options can change between releases. Use the GitHub function for selecting a branch/tag to view the documentation matching your `aws-crossplane-cluster-config-operator` chart version.
+
+The docs below are generated by executing the following command from the repo root directory:
+
+```bash
+make generate-docs
+```
+
+The above `make` target will generate Helm values schema docs by running `schemadocs` like this:
+
+```bash
+schemadocs generate helm/aws-crossplane-cluster-config-operator/values.schema.json -o helm/aws-crossplane-cluster-config-operator/README.md
+```
+
+<!-- DOCS_START -->
+
+### 
+Properties within the `.securityContext` top-level object
+
+| **Property** | **Description** | **More Details** |
+| :----------- | :-------------- | :--------------- |
+| `securityContext.allowPrivilegeEscalation` |**None**|**Type:** `boolean`<br/>|
+| `securityContext.capabilities` |**None**|**Type:** `object`<br/>|
+| `securityContext.capabilities.drop` |**None**|**Type:** `array`<br/>|
+| `securityContext.capabilities.drop[*]` |**None**|**Type:** `string`<br/>|
+| `securityContext.readOnlyRootFilesystem` |**None**|**Type:** `boolean`<br/>|
+| `securityContext.seccompProfile` |**None**|**Type:** `object`<br/>|
+| `securityContext.seccompProfile.type` |**None**|**Type:** `string`<br/>|
+
+### 
+Properties within the `.image` top-level object
+
+| **Property** | **Description** | **More Details** |
+| :----------- | :-------------- | :--------------- |
+| `image.name` |**None**|**Type:** `string`<br/>|
+| `image.pullPolicy` |**None**|**Type:** `string`<br/>|
+| `image.registry` |**None**|**Type:** `string`<br/>|
+| `image.tag` |**None**|**Type:** `string`<br/>|
+
+### 
+Properties within the `.pod` top-level object
+
+| **Property** | **Description** | **More Details** |
+| :----------- | :-------------- | :--------------- |
+| `pod.group` |**None**|**Type:** `object`<br/>|
+| `pod.group.id` |**None**|**Type:** `string`<br/>|
+| `pod.user` |**None**|**Type:** `object`<br/>|
+| `pod.user.id` |**None**|**Type:** `string`<br/>|
+
+### 
+Properties within the `.podSecurityContext` top-level object
+
+| **Property** | **Description** | **More Details** |
+| :----------- | :-------------- | :--------------- |
+| `podSecurityContext.runAsNonRoot` |**None**|**Type:** `boolean`<br/>|
+| `podSecurityContext.seccompProfile` |**None**|**Type:** `object`<br/>|
+| `podSecurityContext.seccompProfile.type` |**None**|**Type:** `string`<br/>|
+
+### 
+Properties within the `.global.podSecurityStandards` object
+
+| **Property** | **Description** | **More Details** |
+| :----------- | :-------------- | :--------------- |
+| `global.podSecurityStandards.enforced` |**None**|**Type:** `boolean`<br/>|
+
+### Other
+
+| **Property** | **Description** | **More Details** |
+| :----------- | :-------------- | :--------------- |
+| `baseDomain` |**None**|**Type:** `string`<br/>|
+| `managementClusterRole` |**None**|**Type:** `string`<br/>|
```

Comment:
I don't see `assumeRole` and `providerRole` listed here. Am I missing something?

Reply:
I generated before I added them, then forgot to generate again

Reply:
Done